q0_linac.py

```diff
@@ -504,9 +535,8 @@ def ds_liquid_level(self, value):
         self.ds_level_pv_obj.put(value)
 
     def fill(self, desired_level=q0_utils.MAX_DS_LL, turn_cavities_off: bool = True):
-        self.ds_liquid_level = desired_level
-        print(f"Setting JT to auto for refill to {desired_level}")
-        caput(self.jtAutoSelectPV, 1, wait=True)
+        print(f"Setting JT to auto for refill to {desired_level} and heater power to 0")
```

Comment:
I'm confused by this print statement. It says 'setting JT to auto,....' but all it does is heater power =0

Reply:
Because I deleted those lines for some reason... good catch...